test/utils/toDayOfWeek.test.js

```diff
@@ -22,8 +23,8 @@ it('Sunday to Friday', () => {
 
 it('Current - To Friday of current week', () => {
   const newDate = toDayOfWeek();
-  const myTempDate = new Date();
-  const distance = 5 - myTempDate.getDay(); // 5 represents Friday
-  myTempDate.setDate(myTempDate.getDate() + distance);
-  expect(newDate).toBe((myTempDate).toISOString().split('T')[0]);
+  const myTempDate = moment().utc();
+  const distance = 5 - myTempDate.day(); // 5 represents Friday
+  myTempDate.date(myTempDate.date() + distance);
+  expect(newDate).toBe(myTempDate.toISOString().split('T')[0]);
```

Comment:
Can we utilize moment's comparison methods here instead of string comparing?

https://momentjs.com/docs/#/query/is-same/